201-dtl-create-vmtemplate/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "existingLabName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of an existing lab where the VM template will be created or updated."
+      }
+    },
+    "existingVMResourceId": {
+      "type": "string",
+      "metadata": {
+        "description": "Resource ID of an existing VM from which the VM template will be created or updated."
+      }
+    },
+    "TemplateName": {

```

Comment:
@ankurmittal: The VM template name must be URL encoded correct (whitespaces replaced with %20), correct?